std/net/curl.d

```diff
@@ -627,6 +627,60 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
```

Comment:
@wilzbach Please note that I am not an experienced d developer. I could do with some help applying these changes, you can contact me in irc #d